docker-compose.mender-gateway.commercial.yml

```diff
@@ -3,7 +3,7 @@ services:
   # mender-gateway
   #
   mender-gateway:
-    image: registry.mender.io/mendersoftware/mender-gateway-qemu-commercial:${MENDER_CLIENT_TAG}
```

Comment:
Seems like this image is following client versioning :monocle_face:

Reply:
Yes, [we talked about it](https://github.com/mendersoftware/integration/pull/2655#discussion_r1812117838).

It is not the client version but the "virtual device version", if you want a call it like that. Basically what used to be the Mender Bundle.

However I will be splitting them this month during the non-server release, so I'll revisit these versions then.